webpack/assets/javascripts/react_app/components/notifications/drawer/NotificationDropdown.js

```diff
@@ -1,24 +1,15 @@
 import React from 'react';
-import { Dropdown, Glyphicon, MenuItem } from 'react-bootstrap';
+import { DropdownKebab, MenuItem } from 'patternfly-react';
 
 const NotificationDropdown = ({ links, id, onClickedLink }) => {
   return (
-    <Dropdown pullRight className="dropdown-kebab-pf" id={id}>
-      <Dropdown.Toggle noCaret bsStyle="link">
-        <Glyphicon bsClass="fa" glyph="ellipsis-v" />
-      </Dropdown.Toggle>
-      <Dropdown.Menu>
-        {
-          links.map((link, i) =>
-            <MenuItem key={i}
-                      id={ `notification-kebab-${i}` }
-                      onClick={ onClickedLink.bind(this, link) }>
-                {link.title}
-            </MenuItem>
-          )
-        }
-      </Dropdown.Menu>
-    </Dropdown>
+    <DropdownKebab pullRight id={id}>
```

Comment:
I am getting this warning about the `id` field:
```
console.error node_modules/prop-types/node_modules/fbjs/lib/warning.js:33
    Warning: Failed prop type: Invalid prop `id` of type `number` supplied to `DropdownKebab`, expected `string`.
        in DropdownKebab
        in NotificationDropdown
```

Need to understand why the `NotificationDropdown` receiving a number id and not a string id.
Anyway, I think we should use a `prop-types` for type validations in our components.

Reply:
I'll take a look at this - I got that same error but I couldn't tell if I was mocking the data correctly.

Reply:
I found the issue is that `patternfly-react` wasn't allowing an integer as the id but it should have. I've submitted a [PR](https://github.com/patternfly/patternfly-react/pull/60) to get this resolved.

Reply:
Cool.
Shouldn't the id suppose to be a string in our case?

Reply:
Based on the [fixtures](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/notifications/notifications.fixtures.js#L31) I was looking at it's an integer.

Reply:
OK, but at the end, it is going to be used as an HTML id attribute so it is going to get converted into string anyway.
I guess that react-bootstrap just wanted to be nice and support numbers because they can easily be converted into strings.

Can convert it into a string before passing to patternfly-react to get rid of this warning?

Reply:
Yeah we can do that. I don't love that solution but I'll add a comment to remove the code once the PR is merged and the package is updated.